test/ToucanRegenBridge.js

```diff
@@ -73,9 +73,11 @@ describe("Token contract", function () {
 	describe("Issue TCO2 tokens", function () {
```

Comment:
should we also add a test that tests the happy path?

Reply:
Need to merge https://github.com/ToucanProtocol/tokenizer/pull/1402 first and update the submodule in this repo before we can test the happy path.

Reply:
gotcha!